### ISSUE TITLE
Refactor training loss to ignore the boundaries 

### ISSUE THREAD

**Issue by monte-flora** [NOAA-National-Severe-Storms-Laboratory/frdd-wofs-cast#2]
Include the border mask in the losses.py
